ci/cleanup.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+if [[ -n "${DEBUG-}" ]]; then
+    set -x
+fi
+
+pushd "$(dirname "$0")"
+
+    UNREAL_PATH="${1:-"$(pwd)/../UnrealEngine"}"
+    BUILD_PROJECT="${2:-NetworkTestProject}"
+
+    PROJECT_ABSOLUTE_PATH="$(pwd)/../${BUILD_PROJECT}"
```

Comment:
Using `..` is classed as an absolute path?

Reply:
It's completely fine. You can also use bash's `realpath` function to collapse it, but it's best to have a script include like svp's workflow/lib/setup.sh that puts into place things like strict mode and cd-into-repo-root-first, among others.